me.proton.Authenticator.yml

```
@@ -0,0 +1,61 @@
+# Based on https://github.com/flathub/me.proton.Pass/blob/a9e4af016548350e0d978297cbb4094ff39c6f24/me.proton.Pass.yml
+
+app-id: me.proton.Authenticator
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: start-proton-authenticator
+finish-args:
+  - --share=ipc
+  - --share=network
+  # - --socket=wayland
+  - --socket=x11
+  - --device=all
+  # DBus-based libsecret
+  - --talk-name=org.freedesktop.secrets
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Disable wayland due to bugs: right click menu disappearing and timers flickering
+  - --env=ELECTRON_OZONE_PLATFORM_HINT=x11
+  - --env=XCURSOR_PATH=~/.icons:/app/share/icons:/icons:/run/host/user-share/icons:/run/host/share/icons
```

```suggestion
  - --env=XCURSOR_PATH=/app/share/icons:/run/host/user-share/icons:/run/host/share/icons
```

/icons does not exist, this has no access to `~/.icons`